packages/language-server/test/completion-items-spec.ts

```diff
@@ -9,13 +9,21 @@ import {
   CompletionItem
 } from "vscode-languageserver";
 import { UI5SemanticModel } from "@ui5-language-assistant/semantic-model-types";
-import { generateModel } from "@ui5-language-assistant/test-utils";
+import {
+  generateModel,
+  GEN_MODEL_TIMEOUT
+} from "@ui5-language-assistant/test-utils";
 
 import { getCompletionItems, computeLSPKind } from "../src/completion-items";
 
-const ui5SemanticModel: UI5SemanticModel = generateModel("1.74.0"); //TODO: use 1.71.x
-
 describe("the UI5 language assistant Code Completion Services", () => {
+  let ui5SemanticModel: UI5SemanticModel;
+  before(async function() {
+    this.timeout(GEN_MODEL_TIMEOUT);
+    //TODO: use 1.71.x
+    ui5SemanticModel = await generateModel("1.74.0");
```

Comment:
switching to 1.71 now breaks tests?

Reply:
I'm not sure but I didn't want to add it to this PR (the comment was already there from before, and I think if we change the version here we should do it in all such tests)

Reply:
o.k, I wonder if we should change test version or not, perhaps it does not matter because we want  to support multiple versions in the future.